src/Microsoft.DotNet.Darc/src/DarcLib/Actions/Local.cs

```diff
@@ -24,7 +25,7 @@ public class Local : ILocal
 
         public Local(string gitPath, ILogger logger)
```

Comment:
>string gitPath, [](start = 21, length = 15)

Not needed anymore

Reply:
Removed.

---
In reply to: [263629746](https://github.com/dotnet/arcade-services/pull/206#discussion_r263629746) [](ancestors = 263629746)